### PR TITLE
jec’s christmas list

### PR DESCRIPTION
**Pull request by cowperthwait**

### CODE REVIEW

**Comment by cowperthwait** [comment]
closes #27

**Comment by kaseybyrne** [comment]
:sheep:

**Comment by AMZod** [comment]
:sheep:

**Comment by rod11** [comment]
:rugby_football:

**Comment by rod11** [comment]
:rose:

**Comment by monicanpm** [comment]
:sheep: it

File: jec/xmaslist.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# All I want for Christmas
+* my two front teeth
```
Comment (monicanpm) [inline jec/xmaslist.md:2]: New pair of Ferragamos

**Pull request merged**